generators/app/index.js

```diff
@@ -8,9 +8,15 @@ module.exports = generators.Base.extend({
     generators.Base.apply(this, arguments);
   },
 
+  copying: function () {
+    this.copy('_gitignore', '.' + 'gitignore');
+    this.copy('_csscomb.json', '.' + 'csscomb.json');
+    this.copy('_eslintrc', '.' + 'eslintrc');
```

Comment:
shouldn't these use the `this.templatePath()` and `this.destinationPath()` functions?